src/client/rooms/dictionary-corner.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import style from './room.scss';
+
+export class DictionaryCorner extends Component {
+    componentWillMount() {
+        this.setState({
+            bestAnswer: {}
+        });
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if(nextProps.bestAnswer) {
+            this.setState({
+                bestAnswer: nextProps.bestAnswer
+            });
+    
+            this.refs['dict-corner'].className += ' ' + style['fadein'];
```

Comment:
Also, using refs like this isn't nice. Please add the className where appropriate in the render function...